src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -607,6 +607,16 @@ CHIP_ERROR DefaultDACVerifier::VerifyNodeOperationalCSRInformation(const ByteSpa
     return CHIP_NO_ERROR;
 }
 
+void DefaultDACVerifier::ValidateDACChainRevocationStatus(const AttestationInfo & info,
+                                                          Callback::Callback<OnAttestationInformationVerification> * onCompletion)
+{
+    AttestationVerificationResult attestationError = AttestationVerificationResult::kSuccess;
+
+    // TODO
```

Comment:
Ah, if we're going to have a default implementation that API consumers might then have to ignore then the logic in the controller makes a bit more sense.  But then shouldn't we do all of our checks and only _then_ call into the provided client callback?  Or have different calls into it for "failed attestation" and "is revoked"?  Because right now, as an API client, how do I tell apart those cases given the APIs we have?

Reply:
Pls review the latest commit.